dotnet/test/Azure.Iot.Operations.Connector.UnitTests/AioCloudEventBuilderTests.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.Operations.Connector.UnitTests;
+
+public class AioCloudEventBuilderTests
+{
+    //TODO: implement tests for AioCloudEventBuilder
```

Comment:
You can either add these tests or delete this file since it currently does nothing